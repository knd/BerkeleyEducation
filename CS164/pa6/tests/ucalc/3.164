1 2/3 / m/s / 1 2/3