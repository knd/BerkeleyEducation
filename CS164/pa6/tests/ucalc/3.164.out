1.0 m^-1 s
